### PR TITLE
Feat docpage

### PR DESCRIPTION
**Pull request by humzahasan**

### CODE REVIEW

File: documentation/doc.css [hunk 1]
```diff
@@ -43,19 +40,74 @@ iframe {
...
 }
 
-.input-box-section {
-  display: flex;
-  justify-content: flex-end;
-  width: 50%;
+/**/
+
+.doc-page {
+  /* background-color: aqua; */
```
Comment (iRohitGaur) [inline documentation/doc.css]: remove commented code

File: documentation/doc.css [hunk 2]
```diff
@@ -43,19 +40,74 @@ iframe {
...
+  display: grid;
+  gap: 1rem;
+  grid-template-areas:
+    "header header header header"
+    "aside main main main"
+    "aside main main main"
+    "aside main main main";
```
Comment (iRohitGaur) [inline documentation/doc.css]: you can do this instead:

header header
aside main

File: library/button/button.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+}
+.btn-outline.btn-success {
+  border: 1px solid rgb(0, 112, 0);
+  color: rgb(0, 112, 0);
+}
+
+.btn-outline.btn-danger {
+  border: 1px solid rgb(207, 19, 19);
+  color: rgb(207, 19, 19);
+}
```
Comment (bhushannaikaroskar) [inline library/button/button.css:71]: Try to use hex values for colors than RGB. Other than that all good

**Review by bhushannaikaroskar — APPROVED** [review]

File: documentation/doc.css (hunk 2 above)
Comment (humzahasan) [reply documentation/doc.css]: Done the changes

File: documentation/doc.css (hunk 1 above)
Comment (humzahasan) [reply documentation/doc.css]: Done the changes

File: library/button/button.css (hunk 3 above)
Comment (humzahasan) [reply library/button/button.css:71]: Yes will do this in next iteration

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by Omkar-Ghate — COMMENTED** [review]
Great work. Good to merge.

**Pull request merged**